### PR TITLE
add entry into JAVA_OPTS to always select log4j2.xml as our logger configuration

### PR DESCRIPTION
**Pull request by stephane-airbyte**

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
* **#35569** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/35569?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stephane-airbyte and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by xiaohansong — APPROVED** [review]

File: airbyte-integrations/bases/base-java/javabase.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ if [[ $IS_CAPTURE_HEAP_DUMP_ON_ERROR = true ]]; then
   fi
 fi
 #30781 - Allocate 32KB for log4j appender buffer to ensure that each line is logged in a single println
-JAVA_OPTS=$JAVA_OPTS" -Dlog4j.encoder.byteBufferSize=32768"
+JAVA_OPTS=$JAVA_OPTS" -Dlog4j.encoder.byteBufferSize=32768 -Dlog4j2.configurationFile=log4j2.xml"
```
Comment (postamar) [inline airbyte-integrations/bases/base-java/javabase.sh:19]: Won't this screw up the logging during integration tests? We want those to be textual.

**Review by postamar — COMMENTED** [review]
Check that this doesn't screw up the logging during integration tests before merging please.

**Review by postamar — APPROVED** [review]
I checked and this seems to work. When building a connector image using the `assemble` gradle task, the log messages printed when we call SPEC as a sanity check are json with this PR and textual on master.

**Pull request merged**